stabilityai/pytorch/inference/buildspec.yml

```diff
@@ -0,0 +1,61 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
+region: &REGION <set-$REGION-in-environment>
+base_framework: &BASE_FRAMEWORK pytorch
+framework: &FRAMEWORK !join [ "stabilityai_", *BASE_FRAMEWORK]
+version: &VERSION 2.0.1
+short_version: &SHORT_VERSION "2.0"
+arch_type: x86
+
+repository_info:
+  inference_repository: &INFERENCE_REPOSITORY
+    image_type: &INFERENCE_IMAGE_TYPE inference
+    root: !join [ *BASE_FRAMEWORK, "/", *INFERENCE_IMAGE_TYPE ]
+    repository_name: &REPOSITORY_NAME !join [pr, "-", "stabilityai", "-", *BASE_FRAMEWORK, "-", *INFERENCE_IMAGE_TYPE]
+    repository: &REPOSITORY !join [ *ACCOUNT_ID, .dkr.ecr., *REGION, .amazonaws.com/, *REPOSITORY_NAME ]
+
+context:
+  inference_context: &INFERENCE_CONTEXT
+    torchserve-ec2-entrypoint:
+      source: docker/build_artifacts/torchserve-ec2-entrypoint.py
+      target: torchserve-ec2-entrypoint.py
+    torchserve-entrypoint:
+      source: docker/build_artifacts/torchserve-entrypoint.py
+      target: torchserve-entrypoint.py
+    config:
+      source: docker/build_artifacts/config.properties
+      target: config.properties
+    deep_learning_container:
+      source: ../../src/deep_learning_container.py
+      target: deep_learning_container.py
+
+images:
+  BuildStabilityaiPytorchCpuPy310InferenceDockerImage:
+    <<: *INFERENCE_REPOSITORY
+    build: &STABILITYAI_PYTORCH_CPU_INFERENCE_PY3 false
+    image_size_baseline: 4900
+    device_type: &DEVICE_TYPE cpu
+    python_version: &DOCKER_PYTHON_VERSION py3
+    tag_python_version: &TAG_PYTHON_VERSION py310
+    os_version: &OS_VERSION ubuntu20.04
+    diffuser_version: &DIFFUSER_VERSION 1.2.3
```

Comment:
should be diffusers, this should be changed

Reply:
Done, thanks!!